cfgov/ask_cfpb/wagtail_hooks.py

```diff
@@ -82,3 +87,42 @@ class MyModelAdminGroup(ModelAdminGroup):
         CategoryModelAdmin,
         SubCategoryModelAdmin,
         NextStepModelAdmin)
+
+
+def editor_js():
+    js_files = [
+        'js/admin/html_editor.js',
+        'js/admin/ask_cfpb_tips.js'
+    ]
+    js_includes = format_html_join(
+        '\n', '<script src="{0}{1}"></script>',
+        ((settings.STATIC_URL, filename) for filename in js_files)
+    )
+
+    return js_includes + format_html(
+        """
+        <script>
+            registerHalloPlugin('editHtmlButton');
+            registerHalloPlugin('answermodule');
+        </script>
+        """
+    )
+
+
+def editor_css():
+    return format_html(
+        '<link rel="stylesheet" href="' +
+        settings.STATIC_URL +
+        'css/question_tips.css">')
+
+
+def whitelister_element_rules():
+    return {
+        'aside': attribute_rule({'class': True}),
+    }
+
+if settings.DEPLOY_ENVIRONMENT == 'build':
```

Comment:
I'm increasingly of the opinion that the use of `DEPLOY_ENVIRONMENT` for this is a bit abusive. Hopefully we can replace these checks with feature flags [soon](https://github.com/cfpb/wagtail-flags/pull/7#issuecomment-290088716).